### PR TITLE
Add empty directory issue to FAQ

### PR DESCRIPTION
**Pull request by emilkovacev**

### CODE REVIEW

**Review by NicholasMy — APPROVED** [review]
Good note to add! We could also potentially add a fix for pulling the submodules without needing to reclone the whole repo, but there are various ways to do that with subtleties that I haven't tested yet.

**Comment by emilkovacev** [comment]
That would be a good addition :) My understanding is `$ git submodules update --init --recursive` generally does the trick, is there something else we should take into consideration?

**Comment by NicholasMy** [comment]
That's what Git's official [documentation](https://git-scm.com/book/en/v2/Git-Tools-Submodules) recommends, so I think it's a safe route to go with.

> To also initialize, fetch and checkout any nested submodules, you can use the foolproof `git submodule update --init --recursive`.

I was concerned that may overwrite manual changes to .gitmodules (e.g., if someone changes the submodule to point to their own branch/fork), but I just tested it and didn't observe any consequences. `--init` is really only necessary for the first run, but it doesn't seem to do any harm to run it again.

Just be mindful of the typo and ensure it's git **submodule** in the final version.

File: readme.md [hunk 1]
```diff
@@ -91,4 +91,17 @@ query: SELECT * FROM "migrations" "migrations" ORDER BY "id" DESC
...
+If this is the case, you would get an error similar to this:
+
+```
+[+] Building 0.0s (0/0)
+unable to prepare context: path "<repo>/<filename>" not found 
+```
+
+We are using `submodules` for this project, which means that both `devU-client/` and `devU-api/` are links to other repos. Make sure you follow the directions for cloning carefully: use the `--recurse-submodules` option when cloning.
+
+**If you already cloned the project**, run the command `git submodule update --init --recursive`, which should pull the appropriate submodules for you.
```
Comment (NicholasMy) [inline readme.md:107]: Looks good!

**Review by NicholasMy — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (emilkovacev) [reply readme.md:107]: Awesome! I'll give you maintainer permissions in the MakeOpenSource DevU repos later today when I get the chance 👍

Comment (emilkovacev) [reply readme.md:107]: Admittedly a little later than I anticipated, but you should now be able to merge PRs across all MOS forks for DevU once you accept the group invite I sent!

Comment (NicholasMy) [reply readme.md:107]: No problem, thanks!

**Pull request merged**